packages/formatter/src/date.ts

```diff
@@ -321,18 +323,21 @@ export function formatDateFriendly(
     timeFirst = false,
     dateOnly = false,
     timeOnly = false,
+    includeSec = false,
```

Comment:
When are you thinking we'd want to show seconds?

Reply:
Maybe question submissions close to the exam end.

Reply:
I think this is a YAGNI. In places like audit logs where we care about second-level precision, we're not going to use "friendly" dates to begin with. This is true in PT already:

<img width="468" alt="Screenshot 2025-07-07 at 12 50 46" src="https://github.com/user-attachments/assets/2508cd29-9a05-4f0a-aa5b-43f843cc524b" />

Reply:
Removed in [37e3baf](https://github.com/PrairieLearn/PrairieLearn/pull/12360/commits/37e3bafeb11a46a71b33365e7176f3da0555b570) and [f002c08](https://github.com/PrairieLearn/PrairieLearn/pull/12360/commits/f002c088ccbf5dd46426c1c38c357f3d4a246041).